<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>471</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButtonWater">
          <property name="text">
           <string>Water</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="radioButtonBranches">
          <property name="text">
           <string>Branches</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>FIO:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="lineFirstName">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLineEdit" name="lineLastName">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5" colspan="2">
         <widget class="QLineEdit" name="linePatronymic">
          <property name="placeholderText">
           <string>Patronymic</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Course:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="lineCourse"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="lineYear">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QLineEdit" name="lineMonth">
          <property name="placeholderText">
           <string>Month</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5" colspan="2">
         <widget class="QLineEdit" name="lineDay">
          <property name="placeholderText">
           <string>Day</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Number:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="lineNumber"/>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Add note</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QPushButton" name="pushButtonRead">
          <property name="text">
           <string>Read from file</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4" colspan="2">
         <widget class="QPushButton" name="pushButtonErase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete note with index:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QLineEdit" name="lineErase"/>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="pushButtonClear">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear journal</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QPushButton" name="pushButtonWrite">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Write to file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="journalElements"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
